view/template/trial.hbs

```diff
@@ -69,6 +69,14 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
+                <div class = "modal-body">
+                    <form id = "check-compliance" method= "GET">
+                        <label>From date:-</label>
```

Comment:
`input` and `label` should be wrapped in [`form-groups`](http://v4-alpha.getbootstrap.com/components/forms/#form-groups).
and `input` should be marked with `form-control`.